### DISCUSSION TITLE
"chezmoi: authentication required" when bootstrapping

### DISCUSSION THREAD

**Discussion by pmolodo** [twpayne/chezmoi#1310] [Q&A] [answered]
I recently tried to bootstrap install and then init / apply my chezmoi repo for a "clean" new windows user.  The install seemed to work, but I get this error when initing:

```
> chezmoi init <my_chemoi_source_repo>
chezmoi: authentication required
```

I've done this several times in the past, though I have made a number of changes to my chezmoi source repo recently - not sure if those are the issue, or it's something about the system I'm installing on, or some change in chezmoi itself.

Any tips on investigating would be helpful - adding a `--debug` to the `init` didn't seem to show anything very useful.

**Comment by twpayne** [comment, marked as answer]
To follow up on this, @pmolodo fixes this in #1315.
